docs/app/pages/showcase/visualizations/organization-chart/organization-chart.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="chart-container" #container>
+    <ux-hierarchy-bar
+        class="hierarchy-bar"
+        [root]="hierarchy"
+        [(selected)]="hierarchyBarSelected"
+        (selectedChange)="onHierarchyBarSelect($event)">
+        <ux-icon name="user-manager" *uxHierarchyBarNodeIcon></ux-icon>
+
+        <div class="search-container" [class.search-container-expanded]="isSearchOpen" uxHierarchyBarRightAddon>
+
+            <ux-toolbar-search [(expanded)]="isSearchOpen" direction="left">
+
+                <input uxToolbarSearchField
+                    #toolbarSearchField
+                    type="text"
+                    class="form-control"
+                    placeholder="Search"
+                    aria-label="Search"
+                    [class.ux-toolbar-search-clear-offset]="query"
+                    [(ngModel)]="query"
+                    (ngModelChange)="search()"
+                    (keydown)="typeaheadKeyService.handleKey($event, typeahead)"
+                    (keydown.enter)="onTypeaheadSelect(highlighted, 'keyboard'); isSearchOpen = false">
+
+                <button *ngIf="query" type="button"
+                    class="btn btn-link btn-icon button-secondary ux-toolbar-search-clear"
+                    (click)="query = ''; toolbarSearchField.focus()">
+                    <ux-icon name="close"></ux-icon>
+                </button>
+
+                <button
+                    uxToolbarSearchButton
+                    type="button"
+                    class="btn btn-link btn-icon button-secondary"
+                    aria-label="Toggle Search"
+                    uxTooltip="Search">
+                    <ux-icon name="search"></ux-icon>
+                </button>
+            </ux-toolbar-search>
+
+            <ux-typeahead
```

Comment:
Could we improve the typeahead item styling:

![image](https://user-images.githubusercontent.com/20795331/72165311-27ef8480-33bf-11ea-961f-20f601e2b437.png)

In MF:

![image](https://user-images.githubusercontent.com/20795331/72165321-2cb43880-33bf-11ea-9be2-010d5caae48d.png)